Mastodon/Scene/MediaPreview/MediaPreviewViewController.swift

```diff
@@ -173,7 +195,13 @@ extension MediaPreviewViewController {
         os_log(.info, log: .debug, "%{public}s[%{public}ld], %{public}s", ((#file as NSString).lastPathComponent), #line, #function)
         dismiss(animated: true, completion: nil)
     }
-    
+
+    @objc private func altButtonPressed(_ sender: UIButton) {
+        os_log(.info, log: .debug, "%{public}s[%{public}ld], %{public}s", ((#file as NSString).lastPathComponent), #line, #function)
+        guard let alt = viewModel.altText else { return }
+        present(AltViewController(alt: alt, sourceView: sender), animated: true)
```

Comment:
More of a taste-question: I'm _a bit_ sceptical when it comes to this log-based debugging. What happened to breakpoints? Please say, that I'm not the only one using them in 2022 😄

In this case I can image, that it's a bit confusing in combination with `guard`, when you look at the console and see that this method was invoced, but, when `viewModel.altText` is `nil`, nothing happens?

Reply:
That makes sense, I can remove. Was just copying the other functions in the file.